helm/hpcc/templates/esp.yaml

```diff
@@ -62,12 +62,12 @@ spec:
 {{ include "hpcc.addConfigMapVolumeMount" . | indent 8 }}
 {{ include "hpcc.addDataVolumeMount" $commonCtx | indent 8 }}
 {{ include "hpcc.addDllVolumeMount" $ | indent 8 }}
-{{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" "storage" "codeSign" ) ) | indent 8 }}
+{{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" "storage" "codeSign" "codeVerify") ) | indent 8 }}
```

Comment:
@shamser @ghalliday I realize it's from an old PR but perhaps we should only add the codeSign category if the $application=eclwatch or whatever application(s) contain your code signing service.  A key like that should be highly restricted.  I doubt we care about overly limiting the public key.

Reply:
@afishbeck I agree that it would be a good idea to restrict just to eclwatch pod.  I'll work on doing  this:https://track.hpccsystems.com/browse/HPCC-25269

Reply:
Great.  FYI if there ever is something that is meant for all ESPs you can use the esp category.